basil/utils/sim/Test.py

```diff
@@ -37,7 +39,7 @@ def socket_test(dut, debug=False):
     port = os.getenv("SIMULATION_PORT", '12345')
 
     if debug:
-        dut._log.setLevel(logging.DEBUG)
```

Comment:
This is this cocotb thing so I am not sure. Better not to touch.

Reply:
Yes, was not sure about this either